doc/README.hbs

```diff
@@ -71,6 +71,21 @@ The example objects contain one key and a single interpolation, but on real
 templates, there can be complex nesting levels and multiple interpolations
 (even many per property).
 
+Default values
+--------------
+--------------
+
+Default values can be provided using the `||` operator. The default value should be JSON encoded.
+In the example below, if the `name` value is not provided, the default value
+`John Doe` will be used instead.
+
+```json
+{
+  "foo": "My name is \{{name || \"John Doe\"}}"
```

Comment:
Should I be able to use a variable at the right of the OR operator? ie `{{name || first_name}}`? If so, this raises the question of whether we should allow any amount of chaining. For instance: `{{ name || first_name || last_name || "John Doe" }}`. What do you think?

Reply:
That sounds great, but I'd prefer to tackle it in a future PR! I'll create an issue so that its not forgotten.